tests/integration/targets/azure_rm_virtualmachine/tasks/azure_test_image_skus.yml

```
@@ -0,0 +1,98 @@
+- name: Set variables
+  ansible.builtin.include_tasks: setup.yml
+
+- name: Set variables
+  ansible.builtin.set_fact:
+    network: 10.42.0.0/24
+    subnet: 10.42.0.0/28
+    vm_name: "vm{{ resource_group | hash('md5') | truncate(6, True, '') }}"
+    network_name: "vnet{{ resource_group | hash('md5') | truncate(6, True, '') }}"
+    subnet_name: "subnet{{ resource_group | hash('md5') | truncate(6, True, '') }}"
+    location: eastus
+    offer: 0001-com-ubuntu-server-focal
+    publisher: Canonical
+
+- name: List available Image SKUs for 0001-com-ubuntu-server-focal
+  azure_rm_imagesku_info:
+    location: "{{ location }}"
+    offer: "{{ offer }}"
+    publisher: "{{ publisher }}"
+  register: available_image_skus_results
+
+- name: Set specific 0001-com-ubuntu-server-focal image version
+  ansible.builtin.set_fact:
+    specific_image_name: "16.04.202104140"
+
+- name: Get the first sku from the available skus list
+  ansible.builtin.set_fact:
+    first_image_sku_option: "{{ available_image_skus_results.available_skus[0] }}"
+
+- name: SETUP | Create virtual network
+  azure_rm_virtualnetwork:
+    resource_group: "{{ resource_group }}"
+    name: "{{ network_name }}"
+    location: "{{ location }}"
+    address_prefixes: "{{ network }}"
+
+- name: SETUP | Create subnet
+  azure_rm_subnet:
+    resource_group: "{{ resource_group }}"
+    name: "{{ subnet_name }}"
+    address_prefix: "{{ subnet }}"
+    virtual_network: "{{ network_name }}"
+
+- name: Create VM with first Image SKU result
+  azure_rm_virtualmachine:
+    resource_group: "{{ resource_group }}"
+    name: "{{ vm_name }}"
+    admin_username: "testuser"
+    ssh_password_enabled: false
```

Please set the open ports, Now , The default 22 port was disallowed by policy！
```suggestion
    ssh_password_enabled: false
    open_ports:
      - 33
```

---

Hi, Fred-sun.
Change applied, Please confirm and merge.